src/Terrain/MapMaster.java

```diff
@@ -310,7 +305,7 @@ public void removeUnit(Unit u)
     {
       if( getLocation(u.x, u.y).getResident() != u )
       {
-        System.out.println("WARNING! Trying to remove a Unit that isn't where he claims to be.");
+        // Assume it's already gone
```

Comment:
Huh. Do we hit this case now?

Reply:
Yeah. When you load or join, the _movement_ now removes the unit from the board. (So that when the "between event" frame is drawn the unit isn't sitting there at the start position)
So when the LOAD/JOIN code comes along, it also tries to `removeUnit`, and it's already off the board.